src/genie/libs/parser/asa/tests/test_show_route.py

```diff
@@ -5,374 +5,62 @@
.1.2/23': {'candidate_default': False, 'active': True, 'route': '10.10.1.2/23', 'source_protocol_codes': 'C', 'source_protocol': 'connected', 'next_hop': {'outgoing_interface_name': {'outside': {'outgoing_interface_name': 'outside'}}}}, '10.122.3.0/24': {'candidate_default': False, 'active': True, 'route': '10.122.3.0/24', 'source_protocol_codes': 'B', 'source_protocol': 'bgp', 'metric': 0, 'route_preference': 20, 'next_hop': {'next_hop_list': {1: {'index': 1, 'next_hop': '172.25.141.2'}}}}, '10.10.1.3/32': {'candidate_default': False, 'active': True, 'route': '10.10.1.3/32', 'source_protocol_codes': 'L', 'source_protocol': 'local', 'next_hop': {'outgoing_interface_name': {'pod2002': {'outgoing_interface_name': 'pod2002'}}}}, '10.10.1.4/32': {'candidate_default': False, 'active': True, 'route': '10.10.1.4/32', 'source_protocol_codes': 'V', 'source_protocol': 'vpn', 'next_hop': {'outgoing_interface_name': {'admin': {'outgoing_interface_name': 'admin'}}}}, '10.10.1.5/32': {'candidate_default': False, 'active': True, 'route': '10.10.1.5/32', 'source_protocol_codes': 'L', 'source_protocol': 'local', 'next_hop': {'outgoing_interface_name': {'pod2500': {'outgoing_interface_name': 'pod2500'}}}}, '10.10.1.6/24': {'candidate_default': False, 'active': True, 'route': '10.10.1.6/24', 'source_protocol_codes': 'C', 'source_protocol': 'connected', 'next_hop': {'outgoing_interface_name': {'pod3000': {'outgoing_interface_name': 'pod3000'}}}}, '10.20.58.64/26': {'candidate_default': False, 'active': True, 'route': '10.20.58.64/26', 'source_protocol_codes': 'E2', 'source_protocol': 'ospf', 'metric': 11, 'route_preference': 110, 'next_hop': {'next_hop_list': {1: {'index': 1, 'next_hop': '172.20.192.3', 'outgoing_interface_name': 'wan1'}}}}, '10.20.2.64/26': {'candidate_default': False, 'active': True, 'route': '10.20.2.64/26', 'source_protocol_codes': 'E2', 'source_protocol': 'ospf', 'metric': 1, 'route_preference': 110, 'next_hop': {'next_hop_list': {1: {'index': 1, 'next_hop': '10.19.1.1', 'outgoing_interface_name': 'wan2'}}}}, '10.30.79.64/26': {'candidate_default': False, 'active': True, 'route': '10.30.79.64/26', 'source_protocol_codes': 'E2', 'source_protocol': 'ospf', 'metric': 11, 'route_preference': 110, 'next_hop': {'next_hop_list': {1: {'index': 1, 'next_hop': '10.20.192.3', 'outgoing_interface_name': 'wan3'}, 2: {'index': 2, 'next_hop': '10.20.192.4', 'outgoing_interface_name': 'wan4'}}}}, '30.20.8.0/23': {'candidate_default': False, 'active': True, 'route': '30.20.8.0/23', 'source_protocol_codes': 'O', 'source_protocol': 'ospf', 'metric': 20, 'route_preference': 110, 'next_hop': {'next_hop_list': {1: {'index': 1, 'next_hop': '172.20.1.1', 'outgoing_interface_name': 'wan5'}}}}, '10.0.0.0/24': {'candidate_default': False, 'active': True, 'route': '10.0.0.0/24', 'source_protocol_codes': 'D', 'source_protocol': 'eigrp', 'metric': 30720, 'route_preference': 90, 'next_hop': {'outgoing_interface_name': {'inside': {'outgoing_interface_name': 'inside'}}}}}}}}}}
```

Comment:
please update the format, a one-line `golden_parsed_output` is hard to read.

we have a tool, which is used to format the parsed output: https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/utils/common.py#L51